app/src/pages/LandingPage.tsx

```diff
@@ -1,41 +1,92 @@
 import React from "react";
 import styles from "./LandingPage.module.scss";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
 
 function LandingPage() {
   return (
     <div className={styles.LandingPage}>
-      <h1>OUR INITIATIVE.</h1>
-      We aim to create a collaborative, multidisciplinary, multisector,
-      public-facing data platform devoted to addressing the health equity
-      implications for vulnerable populations during the COVID-19 pandemic, and
-      develop standardized, evidence-based, best practices for developing
-      socio-culturally responsive guidelines, resources, toolkits, and
-      evaluation metrics for mitigating COVID-19 disparate outcomes for
-      vulnerable populations and to provide technical assistance to health
-      organizations, including state and local jurisdictions’ COVID-19
-      responses.
-      <h1>OUR PROJECT.</h1>
-      Examining the political determinants of health by creating a
-      comprehensive, interactive, public-facing COVID-19 Health Equity Map of
-      the United States and its territories, inclusive of all racial population
-      groups. This project analyzes jurisdictional policies that either
-      exacerbate or alleviate COVID-19 outcomes, comorbidities, resource
-      allocation/management, jurisdictions’ response, and mitigation strategies,
-      testing, and contact tracing, and overall implications for health equity
-      for vulnerable populations. The project allows us to not only understand
-      and share policy research on racial and ethnic and socioeconomic
-      disparities but to understand and share information on the
-      mental/behavioral health equity impact of COVID-19.
-      <h1>OUR IMPACT.</h1>
-      Benchmarking equity to create actionable evidence-based policy responses
-      by engaging with diverse communities across the country to develop policy
-      templates for local, state, and federal policymakers, and develop and
-      publish a state-by-state report card or county-by-county report card based
-      on five to ten objectives/thresholds determined by a national panel of
-      experts under our COVID-19 Health Equity Commission (CHEC). The project
-      creates a special track in HELEN focused on coordinating COVID-19
-      community engagement, education, training, information exchange and
-      dissemination, policy analysis, and advocacy.
+      <Grid container justify="space-around" className={styles.Grid}>
+        <Grid container justify="space-around">
+          <Grid item xs={6} className={styles.BlueLandingSquare}>
+            <Typography variant="h6" className={styles.HomeLogo}>
+              OUR INITIATIVE.
```

Comment:
Nit: I'd drop the periods on the end of these headers. I think it looks weird to have titles with periods